uPortal-webapp/src/main/resources/properties/portal.properties

```diff
@@ -457,6 +457,13 @@ org.apereo.portal.layout.useTabGroups=false
 ##
 org.apereo.portal.layout.useFlyoutMenus=false
 
+
+##
+## Usefull to apply a customized favicon.
+## as example on https://my.univ.edu/favicon.ico set the property to /favicon.ico
+##
+org.apereo.portal.layout.faviconPath=${portal.context}/favicon.ico
```

Comment:
Here I think this property should be commented-out by default (or omitted entirely, if we think it will be a seldom-used option).

The `uPortal.properties`, `global.properties`, _etc._ files are supposed to be optional.  The way it's set up now, I think the Spring context won't load if this property is not defined.

Reply:
Yes if the property isn't set it won't load.

Reply:
as it was set in `portal.properties` you don't need to set it into `uPortal.properties` and `global.properties`... so the property is initialized in the default conf.

On an other side I modified some parts -> the property is needed only if you want to modify the favorite.ico path, in this case you will have only to set `org.apereo.portal.layout.faviconPath` into `uPortal.properties` to your value.